hera.rb

```diff
@@ -0,0 +1,69 @@
+class Hera < Formula
+  desc "Hera: Ewasm virtual machine conforming to the EVMC API"
+  homepage "https://github.com/ewasm/hera"
+
+  stable do
+    url "https://github.com/ewasm/hera/archive/v0.2.4.tar.gz"
+    sha256 "6d1bb30764c892f1aaada26c08add021a981d1d27638d2c9e198ca831544f9d1"
+
+    resource "cmake/cable" do
+      url "https://github.com/ethereum/cable.git", tag: "v0.2.17"
+    end
+
+    resource "evmc" do
+      url "https://github.com/ethereum/evmc.git", tag: "v6.2.2"
+    end
+  end
+
+  head "https://github.com/ewasm/hera.git"
+
+  option "with-debugging", "Build with debugging features and messages"
+  option "without-shared-libraries", "Build libraries as static"
+  option "with-binaryen", "Build with binaryen"
+  option "without-wabt", "Build without wabt"
+  option "with-wavm", "Build with WAVM"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@6"
```

Comment:
Alright, so I had some fun with Hunter, which apparently is a cmake thing that fetches dependencies. Some of the options above require LLVM 6.0, it turns out, so that's included here. It's poured, and not compiled, so apart from the extra disk space it'll occupy I don't see much hazard into including it as is.

That said, the Hunter issue I had was quite unavoidable if I hadn't switched compilers. [These failures are not new](https://github.com/ruslo/hunter/issues/947), and it seems like LLVM 6 will cure them, although I'd like to dig deeper and find out exactly what's causing it.